ConfigSpace/conditions.py

```diff
@@ -176,6 +222,22 @@ def evaluate(self, instantiated_hyperparameters: Hyperparameter) -> bool:
 
         return self._evaluate(evaluations)
 
+    def evaluate_vector(self, instantiated_vector: np.ndarray) -> bool:
+        # Then, check if all parents were passed
+        conditions = self.get_descendant_literal_conditions()
+        for condition in conditions:
+            if condition.parent_vector_id is None:
```

Comment:
No, because the vector id indicates at which position in the vector the parent is located. This cannot be np.NaN